bitshuffle/bitshuffle.py

```diff
@@ -19,31 +21,42 @@
 import re
 import string
 import subprocess
-import tempfile
+from tempfile import mkstemp
+# pylint: disable=ungrouped-imports
+from sys import stderr, stdout, stdin
 
-if __package__ is None or __package__ == '':
-    from errors import *
-else:
-    from .errors import *
+
+try:
+    assert ModuleNotFoundError
```

Comment:
What does this even do? Can you elaborate in a comment maybe?

Reply:
Does assert on a class / function just check that it exists? I was not aware if so, but if this is a widely understood convention I suppose elaboration is not neede comment wise.

Reply:
That's correct, assert will only ever fail if the module or function is undefined. I'll add that as a comment.

The reason it looks so ugly is because I wanted to support use both as an installed module and a linked shellscript; in python2 ModuleNotFoundError is undefined. It's all around more hacky than I'd prefer but it does import errors in every case.